logger/logger.go

```diff
@@ -0,0 +1,39 @@
+package logger
+
+import (
+	log "github.com/hashicorp/go-hclog"
+)
+
+// OurLogger implements an empty logger struct to wrap the go-hclog logger
+type OurLogger struct {
+}
+
+//Logger interface is used to make stuff happen with things in places
+type Logger interface {
+	Error(string, ...interface{})
+}
+
+// Error wraps the go-hclog Error function and logs a separate error for each argument in the error object passed
```

Comment:
I want to try keeping a "discarded attempts" section in my code reviews for now.  I'll flag them all to be removed before merge but I thought we could try it. I'm hoping it helps me be able to keep track of multiple attempts.  Again, may well discard.